tools/c7n_azure/c7n_azure/provider.py

```diff
@@ -29,8 +35,17 @@ class Azure(Provider):
     resource_prefix = 'azure'
     resources = PluginRegistry('%s.resources' % resource_prefix)
     resource_map = ResourceMap
+    region_to_cloud = {
```

Comment:
probably should go in constants with the rest? given its using an import, I think its fine either way tbh